package.json

```diff
@@ -43,7 +43,6 @@
     "passport-reddit": "^0.2.4",
     "q": "1.4.1",
     "rc": "~1.1.2",
-    "redwrap": "0.0.4",
```

Comment:
I got rid of redwrap, because we don't use it except in that RedditController.js thing which we weren't using, and which appears to only output gibberish.